hack/generators/controllers/networking/main.go

```diff
@@ -222,17 +230,20 @@ var inputControllersNeeded = &typesNeeded{
 		RBACVerbs:                         []string{"get", "list", "watch"},
 	},
 	typeNeeded{
-		Group:                             "incubator.ingress-controller.konghq.com",
-		Version:                           "v1alpha1",
-		Kind:                              "KongServiceFacade",
-		PackageImportAlias:                "incubatorv1alpha1",
-		PackageAlias:                      "IncubatorV1Alpha1",
-		Package:                           incubatorv1alpha1,
-		Plural:                            "kongservicefacades",
-		CacheType:                         "KongServiceFacade",
-		NeedsStatusPermissions:            true,
-		ConfigStatusNotificationsEnabled:  true,
-		ProgrammedConditionUpdatesEnabled: true,
+		Group:                            "incubator.ingress-controller.konghq.com",
+		Version:                          "v1alpha1",
+		Kind:                             "KongServiceFacade",
+		PackageImportAlias:               "incubatorv1alpha1",
+		PackageAlias:                     "IncubatorV1Alpha1",
+		Package:                          incubatorv1alpha1,
+		Plural:                           "kongservicefacades",
+		CacheType:                        "KongServiceFacade",
+		NeedsStatusPermissions:           true,
+		ConfigStatusNotificationsEnabled: true,
+		ProgrammedCondition: ProgrammedConditionConfiguration{
+			UpdatesEnabled:       true,
+			CustomUnknownMessage: "KongServiceFacade hasn't been configured in Kong yet. It has to referenced by another resource (e.g. Ingress) to be configured.",
```

Comment:
```suggestion
			CustomUnknownMessage: "Found no references to this resource in Ingress or similar resources.",
```
Wider revamp to use active voice. We can also omit the resource name, it's implied by this appearing in a KongServiceFacade's status.

Reply:
Good point, thanks for the suggestion 🙇